docs/source/index.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to PyDAGMC
+
+PyDAGMC is a Python interface for interacting with DAGMC `.h5m` files
```

Comment:
Can we link the word "DAGMC" to the DAGMC documentation (such as it is) here and throughout?